src/App.tsx

```diff
@@ -15,48 +17,118 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType;
+  isReversed: boolean;
+};
+
+export function getReorderedGoods(
```

Comment:
It's a good practice when you add return type for functions
Example: `function getReorderedGoods(...): string[] {...}`